openstates/nm/legislators.py

```diff
@@ -28,6 +28,9 @@ def scrape_legislator(self, chamber, term, url):
         doc = lxml.html.fromstring(html)
         doc.make_links_absolute(url)
 
+<<<<<<< HEAD
```

Comment:
You can just go in and edit the text. When you merge a file from two different branches that've both been altered from the original, Git does this so that you can see which part of the change was contributed from each.